cmd/rate-limit-test/main.go

```diff
@@ -0,0 +1,68 @@
+package main
```

Comment:
I responded to you in another thread about these tests - for some further considerations
1.  Where should this be given the dependency on local
2. We should probably document the details covered in the issue you have - i.e. formalized docs and such

Reply:
I've removed this file and linked to a copy of it in our private issue should we need to retrieve it later.